packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -20,18 +22,18 @@ import "../funding-rate-store/interfaces/FundingRateStoreInterface.sol";
  * @notice Provides funding rate payment functionality for the Perpetual contract.
  */
 
-contract FundingRateApplier is Lockable {
-    using SafeMath for int256;
-    using SafeMath for uint256;
+abstract contract FundingRateApplier is Testable, Lockable {
     using FixedPoint for FixedPoint.Unsigned;
+    using FixedPoint for FixedPoint.Signed;
     using SafeERC20 for IERC20;
+    using SafeMath for uint256;
 
     /****************************************
      * FUNDING RATE APPLIER DATA STRUCTURES *
      ****************************************/
 
     // Points to financial product-related contracts like the funding rate store.
-    FinderInterface public fpFinder;
+    FinderInterface private fpFinder;
```

Comment:
I think its worth making a comment here that this method is `private` because its assumed that the inheriting contract is also inheriting FeePayer which makes the `finder` `public`

Reply:
Good point. Will do.

Reply:
Done.